processing/src/main/java/org/apache/druid/query/groupby/GroupByQuery.java

```diff
@@ -249,7 +249,6 @@ private GroupByQuery(
     return subtotalsSpec;
   }
 
-  @Override
```

Comment:
Why this change, `Query.java` provides a default implementation, so doesn't it `@Override`?

Reply:
This was a change from removing the "old rewrite mode" initially, but now that it's added back in, so I put the overrides back